### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mfmurray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,174 @@
-# RowndAngular
+# Angular
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.1.1.
+## Installation
 
-## Development server
+run `npm install @rownd/angular` or `yarn add @rownd/react`
```
Comment (mhamann) [inline README.md]: yarn should be `@rownd/angular`

File: README.md [hunk 2]
```diff
@@ -1,27 +1,174 @@
...
+```
 
-## Code scaffolding
+## Usage
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+The library provides a Rownd Module and Dependency Injection (Service).
 
-## Build
+In the main app.module.ts file, add the Rownd Module. Please include the @ngrx/store Module as well.
```
Comment (mhamann) [inline README.md:28]: Do devs need to install `@ngrx/store` first or is that a default?

Comment (mfmurray) [reply README.md:28]: It's not a default when you install a basic angular app. I can only get the sdk to work if @ngrx/store is added to the main file

**Review by mhamann — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mfmurray) [reply README.md]: haha wonder where I got that from...

Comment (mfmurray) [reply README.md]: fixed

**Pull request merged**